mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/modules/CassandraAclModule.java

```diff
@@ -36,6 +36,7 @@
         .table(CassandraACLTable.TABLE_NAME)
         .comment("Holds mailbox ACLs")
         .options(options -> options
+            .compressionOptions(SchemaBuilder.lz4().withChunkLengthInKb(4))
```

Comment:
You can extract `SchemaBuilder.lz4().withChunkLengthInKb(4)` into a constant `COMPRESSION_OPTION` into `CassandraModule` ?

Reply:
For sure, but not named like this as it obfuscate the intent.

Reply:
It's a case where I don't think having a constant adds anything.
It rather hide the intent instead of making it more obvious.